api/server.go

```diff
@@ -68,10 +69,8 @@ func NewServer(ctx context.Context, ts tserv.Threadservice, conf Config) (*Serve
 		cancel:  cancel,
 	}
 
-	if conf.Addr == "" {
-		conf.Addr = "0.0.0.0:9090"
-	}
-	listener, err := net.Listen("tcp", conf.Addr)
+	addr := util.TCPAddrFromMultiAddr(conf.Addr, "127.0.0.1:9090")
+	listener, err := net.Listen("tcp", addr)
```

Comment:
The switch between `0.0.0.0` and `127.0.0.1` was intentional?
It's done in other places too, this questions count for all places :)

Reply:
For the API, loopback seems like a more reasonable default.